invoice/bill.go

```diff
@@ -272,18 +276,19 @@ func (b *Bill) drawBillablesTable(headers []string, billables []BillableItem, wi
 	// Draw Tax
 	b.pdf.Ln(4)
 	b.drawBlanks(billables, widths)
-	b.textFormat(widths[len(widths)-2], 4, "Tax", "1", 0, "R", true, 0, "")
-	b.textFormat(widths[len(widths)-1], 4, "0", "1", 0, "R", true, 0, "")
+	taxText := billables[0].Currency + " " + niceFloatStr(tax)
+	b.textFormat(widths[len(widths)-2], 4, "GST", "1", 0, "R", true, 0, "")
```

Comment:
```suggestion
	b.textFormat(widths[len(widths)-2], 4, "Tax", "1", 0, "R", true, 0, "")
```

GST is a region-specific tax for certain countries. Let's stay with generic "Tax"

Reply:
Alternatively, make this configurable in the `default_tax` block in the config.